fotoobo/tools/fgt/get.py

```diff
@@ -17,6 +17,33 @@
 log = logging.getLogger("fotoobo")
 
 
+def api_get(
```

Comment:
1. Why do you define this function here under tools?
2. Why do you use this function in other code?
I would suggest you move this logic to the Infrastructure Layer (`fortinet/fortigate.py`) and use this one from the other functions, and also this function. Otherwise we create inter-layer dependencies (-> What happens with the other functions, if you change the signature of this one, because CLI frontend requires it?)

Reply:
I refacored it to have the api_get() methos as a low level method.
I alos think this is better